smdebug/core/state_store.py

```diff
@@ -26,7 +27,7 @@ def _get_checkpoint_files_in_dir(self, cp_dir):
         checkpoint_files = []
         for child, _, files in os.walk(cp_dir):
             for file in files:
-                if file != METADATA_FILENAME:
+                if file != METADATA_FILENAME and file != METADATA_FILENAME_S3_UPLOADED:
```

Comment:
remove this not needed.

Reply:
sure.

Reply:
removing this fails this test - https://github.com/awslabs/sagemaker-debugger/pull/310/files#diff-9ec1a0fff79c490c65c2ae46f7e51d09R39